velox/docs/monitoring/stats.rst

```diff
@@ -26,7 +27,7 @@ Memory Arbitration
      - Type
      - Description
    * - memoryReclaimCount
-     - kNone
+     - Quantity
```

Comment:
Shall we update the code definition as well?

Reply:
use `<empty>` according to this comment https://github.com/facebookincubator/velox/pull/9255#discussion_r1539723555